partitioned-table.md

```diff
@@ -404,6 +404,44 @@ test> select * from t;
 3 rows in set (0.01 sec)
 ```
 
+New in v7.1 is the support of a DEFAULT list partition, which acts as a 'catch-all' partition, where all rows that does not match the defined sets of values will be placed.
+
+Enable this feature with `set tidb_enable_default_list_partition = ON`, it is OFF by default, since it is not supported by MySQL.
```

Comment:
```suggestion
Before using the default list partition feature, you need to set the `tidb_enable_default_list_partition` system variable to `ON`, which is `OFF` by default because it is not supported in MySQL.
```